cmd/nerdctl/container.go

```diff
@@ -42,6 +42,7 @@ func newContainerCommand() *cobra.Command {
 		newStopCommand(),
 		newStartCommand(),
 		newRestartCommand(),
+		newHistoryCommand(),
```

Comment:
This has to be under `nerdctl image`, not `nerdctl container`.

Also please update README.md

Reply:
done,thank you! Happy spring  Festival

Reply:
Xinnian kuaile ㊗️ , could you rebase with the master to pass the CI?

Reply:
Ping @ningmingxiao 🙏